app/Http/Controllers/SettingController.php

```diff
@@ -123,4 +124,20 @@ public function cleanupImages(Request $request, ImageService $imageService)
 
         return redirect('/settings/maintenance#image-cleanup')->withInput();
     }
+
+    /**
+     * Action to send a test e-mail to the current user.
+     * @param Request $request
+     * @param User $user
+     * @return \Illuminate\Http\RedirectResponse|\Illuminate\Routing\Redirector
+     */
+    public function sendTestEmail(Request $request)
+    {
+        $this->checkPermission('settings-manage');
+
+        user()->notify(new TestEmail());
+        $this->showSuccessNotification(trans('settings.maint_send_test_email_success', ['address' => user()->email]));
+
+        return redirect('/settings/maintenance#image-cleanup')->withInput();
```

Comment:
I just noticed I forgot to change the anchor here after copy & paste from above :)